### PR TITLE
issue: 1006082 Update VMA_SPEC name stac to multi_ring_latency

### PR DESCRIPTION
**Pull request by DanielLibenson**
Updated stac spec name to multi_ring_latency name (VMA_SPEC).

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1383/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/921/ for details (Mellanox internal link).

File: README.txt [hunk 1]
```diff
@@ -278,8 +278,11 @@ latency
      VMA_THREAD_MODE = Single                 (default: Multi spin lock)
      VMA_MEM_ALLOC_TYPE = 2                   (default: 1 (Contig Pages))
     Example: VMA_SPEC=latency
-stac
-Optimized profile for STAC-N Benchmark
+
+multi_ring_latency
+     Optimized for use cases that are keen on latency using distinctive send socket and receive socket.
```
Comment (OphirMunk) [inline README.txt]: Suggesting
```
     Optimized for use cases that are keen on latency where two applications communicate using send-only and receive-only TCP sockets
```

**Comment by OphirMunk** [comment]
Merged #324

**Pull request closed without merging**